libnavigation-base/src/main/java/com/mapbox/navigation/base/formatter/UnitType.kt

```diff
@@ -0,0 +1,18 @@
+package com.mapbox.navigation.base.formatter
+
+/**
+ * Unit type for voice and visual information. Note that this won't change other results such as
+ * raw distance measurements which will always be returned in meters.
+ */
+enum class UnitType {
```

Comment:
Now SDK would provide both `VoiceUnit.Type` string def and `UnitType` enum.

If we're confident in using an enum (if we know we won't add any other properties here), then I think we should just transform `VoiceUnit` into an enum and make it public (it currently lives in an `internal` package even though it's used inthe public API surface which is another issue).

Reply:
ok, let's do it. But I think we should have `UnitType` enum, not `VoiceUnit`, it is used not only for voice